integration/update_test.go

```diff
@@ -466,6 +466,58 @@ func TestUpdateSet(t *testing.T) {
 	}
 }
 
+func TestUpdateTimestamp(t *testing.T) {
+	t.Parallel()
+
+	// store the current timestamp with $currentDate operator;
+	t.Run("currentDateReadBack", func(t *testing.T) {
+		maxDifference := time.Duration(2 * time.Second)
+		now := time.Now()
+		nowTimestamp := primitive.Timestamp{T: uint32(now.Unix()), I: uint32(0)}
+		id := "string-empty"
+
+		update := bson.D{{"$currentDate", bson.D{{"value", bson.D{{"$type", "timestamp"}}}}}}
```

Comment:
Let's move this line next to the actual update operation right before its usage.

Reply:
> Let's move this line next to the actual update operation right before its usage.

It seems to me that it's a code polishing, which is discussed the previous week. But ok, I'll fix it now...